app/manifest.yml

```diff
@@ -72,6 +72,7 @@ permissions:
     - storage:app
     - write:build-info:jira
     - delete:build-info:jira
+    - read:deployment-info:jira
```

Comment:
Why do we need to read deployments in Jira now?

Reply:
getgatingstatus is doing a GET call on deployments and is currently failing with a 401